app/src/main/java/org/p2p/wallet/home/HomeModule.kt

```diff
@@ -108,6 +110,30 @@ object HomeModule : InjectionModule {
                 renBtcAnalytics = get()
             )
         }
+
+        factory<ReceiveTokenContract.Presenter> { (token: Token.Active) ->
+            ReceiveTokenPresenter(
+                defaultToken = token,
+                qrCodeInteractor = get(),
+                usernameInteractor = get(),
+                tokenKeyProvider = get(),
+                receiveAnalytics = get()
+            )
+        }
+
+        factoryOf(::TokenListPresenter) bind TokenListContract.Presenter::class
+        factoryOf(::ReceiveRenBtcPresenter) bind ReceiveRenBtcContract.Presenter::class
+
+        includes(createSendModule())
+    }
+
+    private fun createSendModule(): Module = module {
+        factory<SelectTokenContract.Presenter> { (tokens: List<Token>) ->
+            SelectTokenPresenter(tokens)
+        }
+        factory<SearchContract.Presenter> { (usernames: List<SearchResult>) ->
+            SearchPresenter(usernames = usernames, searchInteractor = get(), usernameDomainFeatureToggle = get())
+        }
```

Comment:
maybe it's time to create a separate module for Send feature?

Reply:
you want to extract all the send screens to a separate module SendModule
or only new ones so it would be NewSendModule?